### ISSUE TITLE
Centered tabs

### ISSUE THREAD

**Issue by remicastaing** [angular/material#1992]
Is there a way to center tabs like in G+?

![capture d ecran 2015-03-21 10 06 47](https://cloud.githubusercontent.com/assets/1715952/6764503/1b54c12a-cfb2-11e4-8639-29853f05c70d.jpg)

**Comment by bonatoc** [comment]
On line 48 of [https://github.com/angular/material/blob/master/src/components/tabs/tabs.scss](https://github.com/angular/material/blob/master/src/components/tabs/tabs.scss),
it looks like it could be an (ng-)attribute of md-tabs.

I have no time to try it right now, but I would like if you give me some feedback if you test it, thank you.

**Comment by bonatoc** [comment]
I think you can "only" stretch them, [see the doc](https://material.angularjs.org/#/api/material.components.tabs/directive/mdTabs).

So maybe you can place them into another fixed-width container that is centered (think flexbox), so the tabs do not stretch on all the topnav's width, like in the example you are showing.

**Comment by bonatoc** [comment]
OK I think I have it. [See this codepen](http://codepen.io/bonatoc/pen/GgzgXq) and please let me know.

**Comment by remicastaing** [comment]
It works well. Thank you, @bonatoc for taking the time.

I adapted your code a little for small sized screen: [codepen](http://codepen.io/remicastaing/pen/VYgvmx).

**Comment by remicastaing** [comment]
Should we keep this issue open and tagged as improvement?

**Comment by bonatoc** [comment]
I'm not sure. 

We could contribute by making a **md-centered-tabs** attribute following the "wrapped-inside-centered-container" and request a pull.

But I think the **md-stretch-tabs** attribute already does the job. 
Frameworks are there to be hacked, worked with. 
We can't expect them to have all the options, and I don't think it is the way to go. You end up overbloating it (see Bootstrap...).

**Comment by bonatoc** [comment]
Maybe you could tagged the issue as improvement **for the demos** ?

Many devs would like the centered tabs option displayed as an example.

Can we fork the docs a make a pull request on the demos?

**Comment by remicastaing** [comment]
I tried to write a simple md-centered-tabs directive, but it is not simple. I gave up.

Your solution, @bonatoc, should be kept somewhere. She's very useful.

**Comment by robertmesserle** [comment]
Support for centered tabs is on my list, and I will be hopefully getting to this today.

**Comment by remicastaing** [comment]
Awesome @robertmesserle. And again, thanks @bonatoc.

**Comment by bonatoc** [comment]
Awesome @robertmesserle 

@remicastaing , you're welcome. I was on the same issue myself.

**Comment by super-coder** [comment]
both codepens don't work for me

so can i use centered tabs?

**Comment by bonatoc** [comment]
@super-coder — if you're super, please provide more info: version, specific problem?

**Comment by super-coder** [comment]
![image](https://cloud.githubusercontent.com/assets/2155554/12327618/a60843b0-bae8-11e5-81ff-ca667a702839.png)

![image](https://cloud.githubusercontent.com/assets/2155554/12327624/af609a20-bae8-11e5-898c-f11e044908ff.png)

**Comment by prasad47** [comment]
md-center-tabs="true"  did the trick just like that FYI

**Comment by super-coder** [comment]
i submitted an issue https://github.com/angular/material/issues/7695, tabs are hidden after reload on IE on XS screen

**Comment by bonatoc** [comment]
@super-coder — Please provide IE version.

**Comment by super-coder** [comment]
@bonatoc what do you mean? you don't understand how code doesn't work in IE?? There is no IE version

**Comment by bonatoc** [comment]
"There is no IE version" - I beg your pardon?

**Comment by super-coder** [comment]
@bonatoc you can see the screen in https://github.com/angular/material/issues/7695

**Comment by bonatoc** [comment]
Like tagged: "need more info".

**Comment by super-coder** [comment]
added more info there

**Comment by jonathanmeguira** [comment]
just for you to know this works wonders as well 

<md-center-tabs>
  <md-tabs md-dynamic-height md-border-bottom>
   <md-tabs></md-tabs>
</md-tabs>
<md-center-tabs>

**Comment by jonathanmeguira** [comment]
here's the working code 
`                    <md-center-tabs>
                    <md-tabs md-cnter-tabs="true" md-dynamic-height md-border-bottom>
      <md-tab label="Policy Dashboard">
        <md-content class="md-padding">
          <h1 class="md-display-2">Policy Dashboard</h1>
          <p>{{lorem}}</p>
        </md-content>
      </md-tab>
      <md-tab label="What is this policy?">
        <md-content class="md-padding">
          <h1 class="md-display-2">What is this policy?</h1>
          <p>{{lorem}}</p>
        </md-content>
      </md-tab>
      <md-tab label="Who uses this Policy?">
        <md-content class="md-padding">
          <h1 class="md-display-2">Who uses this Policy?</h1>
          <p>{{lorem}}</p>
        </md-content>
      </md-tab>
    </md-tabs>
                        </md-center-tabs>`

**Comment by jonathanmeguira** [comment]
replace ({lorem}} by the content of your choice of course

**Comment by TXZdream** [comment]
Just add md-center-tabs="true" in md-tabs is OK, jonathanmeguira's code doesn't work because he spell mistake 'center'.

**Comment by jonathanmeguira** [comment]
my bad @TXZdream

**Comment by Zwilla** [comment]
open angular.material.js and search for: material.components.tabs
there you will find everything about the tabs and how they work.

**Source Code**
* https://github.com/angular/material/tree/master/src/components/tabs

**build***
* https://ajax.googleapis.com/ajax/libs/angular_material/1.1.4/angular-material.js

**EXAMPLE:**
`<md-tabs mdTabScroll md-align-tabs="top" md-selected="0" md-dynamic-height md-border-bottom layout-fill md-stretch-tabs="always" md-center-tabs md-swipe-content>`
